internal/pkg/deploy/env.go

```diff
@@ -17,6 +18,17 @@ const (
 	LatestEnvTemplateVersion = "v1.10.0"
 )
 
+// LatestAvailableFeatures is a list of latest available feature names.
+var LatestAvailableFeatures = []string{template.ALBFeatureName, template.EFSFeatureName, template.NATFeatureName, template.InternalALBFeatureName}
```

Comment:
> Is there a benefit in having this list separate than a template.AvailableEnvFeatures? (basically having everything in the template pkg)

Implementation-wise, not much. I just thought `AvailableEnvFeatures` are features available to use in a deployed env stack, hence `deploy`. Plust it's close to where `LatestEnvTemplateVersion` lives.  

Though it can also be argued that `template` is the right package since `template` determines the template for a deployed env. This also comes with the benefit of easier to maintain than the other route. Changing 👍🏼